src/components/aktivitetskrav/vurdering/AvventFristDato.tsx

```diff
@@ -10,17 +11,17 @@ export const avventerFristDatoField = "fristDato";
 
 export const AvventFristDato = () => {
   const now = new Date();
-  const today = dayjs(now);
-  const inTwoMonths = today.add(2, "months");
+  const inTwoMonths = dayjs(now).add(2, "months").toDate();
```

Comment:
Forstår jeg det sånn at dersom de velger å avventer mer enn 2 mnd så er det ugyldig / for lenge?